packages/job-manager/src/batch/pool-load-generator.spec.ts

```diff
@@ -22,7 +24,7 @@ describe(PoolLoadGenerator, () => {
                     activeToRunningTasksRatio: activeToRunningTasksRatio,
                     addTasksIntervalInSeconds: 15,
                     maxWallClockTimeInHours: 1,
-                };
+                } as JobManagerConfig;
```

Comment:
can you add a value for "sendNotificationTasksCount" instead of casting?, and then check the value in the UTs something like: **expect(poolLoadGenerator.sendNotificationTasksCount).toEqual(x);**

Reply:
nope. prefer to only set the properties that we need. this way you dont have to update this test whenever a new property is added.